Makefile

```diff
@@ -119,38 +119,39 @@ all:	$(SDCARD_DIR)/MEGA65.D81 $(BINDIR)/mega65r1.mcs $(BINDIR)/nexys4.mcs $(BIND
 
 freezer_files: $(FREEZER_FILES)
 
+SMU= \
```

Comment:
Maybe rename this macro SUBMODULEUPDATE, so that its more self-evident when it is being used later in the Makefile?

Reply:
Okidoke